runtime/rastrace/trcengine.c

```diff
@@ -463,7 +465,6 @@ J9VMDllMain(J9JavaVM *vm, IDATA stage, void *reserved)
 		break;
 
 	case VM_INITIALIZATION_COMPLETE:
-		tempThr = UT_THREAD_FROM_VM_THREAD(vm->mainThread);
```

Comment:
This is not a part of the build error fix, but `tempThr` is never used in the `VM_INITIALIZATION_COMPLETE` path.